config.go

```diff
@@ -119,6 +119,13 @@ func LoadConfiguration() (*configuration.Configuration, error) {
 		Hash:  os.Getenv(GenesisBlockHashEnv),
 	}
 
+	config.Network = &types.NetworkIdentifier{
+		Blockchain: blockchain,
+		Network:    networkValue,
+	}
+	config.GenesisBlockIdentifier = genesisBlockHash
+	config.ChainConfig = params.MainnetChainConfig
```

Comment:
Do we still need mainnet as default?

Reply:
was fixed in 8c8a2b85c2088cbbe5011e4d9a2d860a3a0bab27